sdk/spring/azure-spring-cloud-eventhubs-stream-binder/pom.xml

```diff
@@ -38,7 +38,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;dependency} -->
+      <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs-checkpointstore-blob;dependency} -->
```

Comment:
1.3.0 or 1.3.1?

Reply:
1.3.0 is correct for spring package. We haven't released 1.3.1 yet.